pkg/search/controller.go

```diff
@@ -352,7 +352,7 @@ func (c *Controller) getRegistryBackendHandler(cluster string, matchedRegistries
 }
 
 var clusterDynamicClientBuilder = func(cluster string, controlPlaneClient client.Client) (*util.DynamicClusterClient, error) {
-	return util.NewClusterDynamicClientSet(cluster, controlPlaneClient)
+	return util.NewClusterDynamicClientSet(cluster, controlPlaneClient, nil)
```

Comment:
Why pass nil here?

Reply:
karmada-search didn't provide `--cluster-api-qps' and '--cluster-api-burst` flags, currently its qps and burst are always 5 and 10, i.e. default value from `rest.Config`.
maybe later we will consider add these flags to it, but in this PR I think we could keep it untouched.

Reply:
Makes sense. But doesn't harm to add a comment to explain why pass nil here :)